icubam/www/handlers/update.py

```diff
@@ -51,6 +48,17 @@ async def get(self):
     self.set_secure_cookie(self.COOKIE, user_token)
     self.render('update_form.html', **data)
 
+
+class UpdateBedCountsHandler(base.BaseHandler):
+  """Register new bedcounts coming from the form."""
+
+  ROUTE = updater.Updater.POST_ROUTE
+
+  def initialize(self, config, db_factory, queue):
+    super().initialize(config, db_factory)
+    self.queue = queue
+    self.updater = updater.Updater(self.config, self.db)
+
   @tornado.web.authenticated
```

Comment:
How is this going to work, since `get_current_user` always returns `None`?

Reply:
Sorry, I got tricked by the collapsed diff. I understand now, please ignore my question.